a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -211,9 +218,17 @@ def get_delete_load_balancer_flow(self, lb, deleteCompute):
             requires=a10constants.VTHUNDER))
         return (delete_LB_flow, store)
 
-    def get_new_lb_networking_subflow(self, topology):
+    def get_new_lb_networking_subflow(self, load_balancer_id, topology):
         """Subflow to setup networking for amphora"""
-
+        lb = self._lb_repo.get(db_apis.get_session(),
+                               id=load_balancer_id)
+        vthunder = self._vthunder_repo.get_vthunder_by_project_id(db_apis.get_session(),
+                                                                  lb.project_id)
+        lb_exists_flag = False
+        if vthunder:
+            lb_exists_flag = self.loadbalancer_repo.get_lb_exists_flag(db_apis.get_session(),
```

Comment:
This is just minor comment:
Since lb_exists_flag is only used for AmphoraePostVIPPlug task, can we move these check in AmphoraePostVIPPlug()?
so, we keep logic in tasks and get flow functions only add tasks in flow.

Reply:
Done